### PR TITLE
Helm Charts

### PR DESCRIPTION
**Pull request by rvowles**
These are the helm charts that are designed to bring up FeatureHub in a cluster.

### CODE REVIEW

File: helm/featurehub/templates/edge-config.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: edge-config
+data:
+  application.properties: |-
+    server.port=8553
+    maxSlots=60
+    update.pool-size=30
```
Comment (mpluhar) [inline helm/featurehub/templates/edge-config.yaml]: I think you would expect those values in the values.yaml.

File: helm/featurehub/templates/mr-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "featurehub.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ include "featurehub.managementRepository.name" . }}
```
Comment (mpluhar) [inline helm/featurehub/templates/mr-deployment.yaml:29]: Can you define memory and cpu limits?

Comment (rvowles) [reply helm/featurehub/templates/mr-deployment.yaml:29]: there is a resources section to include from the yaml, but its overall, so it should probably be per app

File: helm/featurehub/templates/edge-config.yaml (hunk 1 above)
Comment (rvowles) [reply helm/featurehub/templates/edge-config.yaml]: true

File: helm/featurehub/templates/mr-deployment.yaml (hunk 2 above)
Comment (rvowles) [reply helm/featurehub/templates/mr-deployment.yaml:29]: i.e. they all get the same resource allocated if its defined

**Pull request merged**